### PR TITLE
Sonnar & Sonnar_emit will return TVMaze ID if present

### PR DESCRIPTION
**Pull request by liiight**
Sonnar & Sonnar_emit will return TVMaze ID if present

### CODE REVIEW

File: flexget/plugins/input/sonarr_emit.py [hunk 1]
```diff
@@ -115,15 +116,8 @@ def on_task_input(self, task, config):
...
-                        log.info("    URL: %s" % entry["url"])
-                        log.info("    Show name: %s" % entry["series_name"])
-                        log.info("    TVDB ID: %s" % entry["tvdb_id"])
-                        log.info("    TVRAGE ID: %s" % entry["tvrage_id"])
-                        log.info("    Season: %s" % entry["series_season"])
-                        log.info("    Episode: %s" % entry["series_episode"])
-                        log.info("    Series ID: %s" % entry["series_id"])
-                    continue
+                        for key, value in entry.items():
+                            log.info('     %s: %s' % (key.capitalize(), value))
```
Comment (gazpachoking) [inline flexget/plugins/input/sonarr_emit.py]: I don't think this should be info level. verbose, or even debug is probably better don't you think?

Comment (liiight) [inline flexget/plugins/input/sonarr_emit.py]: Well it's info in all other Plugins I saw, this is test mode remember. 
If you want I'll make it verbose but I don't think that or debug is needed.

Comment (gazpachoking) [inline flexget/plugins/input/sonarr_emit.py]: Oh, just for test mode. I think verbose is appropriate then, I suspect you'd be running a --test manually anyway, so you'll be in verbose mode by default.

Comment (liiight) [inline flexget/plugins/input/sonarr_emit.py]: I got no problems with that other than inconsistency with other plugins. But I can always change those as well when the opportunity arises I guess.

**Pull request merged**